docs/contracts/node-operators-registry.md

```diff
@@ -14,126 +39,306 @@ Returns the rewards distribution proportional to the effective stake for each no
+function getSigningKey(uint256 _nodeOperatorId, uint256 _index) returns (
+    bytes key,
+    bytes depositSignature,
+    bool used
 )
 ```
 
-#### Parameters:
+#### Parameters
 
-| Name           | Type      | Description                        |
-| -------------- | --------- | ---------------------------------- |
-| `_operator_id` | `uint256` | Node Operator id                   |
-| `_index`       | `uint256` | Index of the key, starting with 0d |
+| Name              | Type      | Description                       |
+| ----------------- | --------- | --------------------------------- |
+| `_nodeOperatorId` | `uint256` | Node Operator id                  |
+| `_index`          | `uint256` | Index of the key, starting with 0 |
 
-#### Returns:
+#### Returns
 
 | Name               | Type    | Description                                           |
 | ------------------ | ------- | ----------------------------------------------------- |
 | `key`              | `bytes` | Key                                                   |
 | `depositSignature` | `bytes` | Signature needed for a `depositContract.deposit` call |
 | `used`             | `bool`  | Flag indication if the key was used in the staking    |
 
+### getSigningKeys()
+
+Returns `_limit` signing keys of the node operator.
+
+```sol
+function getSigningKey(uint256 _nodeOperatorId, uint256 _offset, uint256 _limit) returns (
+    bytes pubkeys,
+    bytes signatures,
+    bool[] used
+)
+```
+
+#### Parameters
+
+| Name              | Type      | Description                        |
+| ----------------- | --------- | ---------------------------------- |
+| `_nodeOperatorId` | `uint256` | Node Operator id                   |
+| `_offset`         | `uint256` | Offset of the key, starting with 0 |
+| `_limit`          | `uint256` | Number of keys to return           |
+
+#### Returns
+
+| Name         | Type     | Description                                                                             |
+| ------------ | -------- | --------------------------------------------------------------------------------------- |
+| `pubkeys`    | `bytes`  | Keys concatenated into the bytes batch                                                  |
+| `signatures` | `bytes`  | Signatures concatenated into the bytes batch needed for a deposit_contract.deposit call |
+| `used`       | `bool[]` | Array of flags indicated if the key was used in the staking                             |
+
 ### getNodeOperatorsCount()
 
-Returns total number of node operators
+Returns the total number of node operators.
 
 ```sol
 function getNodeOperatorsCount() returns (uint256)
 ```
 
 ### getKeysOpIndex()
-Returns a monotonically increasing counter that gets incremented when any of the following happens:
-1. a Node Operator's key(s) is added
-2. a Node Operator's key(s) is removed
-3. a Node Operator's approved keys limit is changed
-4. a Node Operator was activated/deactivated
+
+:::note
+DEPRECATED use getNonce() instead
```

Comment:
It's arguable, but I removed all entries for deprecated methods in lido.md. They create more noise than value, imho.